src/hooks/useApproveCallback.ts

```diff
@@ -2,7 +2,7 @@ import { MaxUint256 } from '@ethersproject/constants'
 import { TransactionResponse } from '@ethersproject/providers'
 import { Trade, TokenAmount, CurrencyAmount, ETHER } from '@uniswap/sdk'
 import { useCallback, useMemo } from 'react'
-import { ROUTER_ADDRESS } from '../constants'
+import { ROUTER_ADDRESS } from '@/constants'
```

Comment:
this part should work with:
import { ROUTER_ADDRESS } from 'constants/index'

see https://github.com/Uniswap/uniswap-interface/pull/1186/files#diff-aa2d8ce482205a06ae169fb70815e4de362af6a187eb5276a393386091ebd58aR12